### ISSUE TITLE
Castle Of The Winds tile renderer

### ISSUE THREAD

**Issue by Lokathor** [Lokathor/thorium#11]
We want a renderer that can use the [Castle of The Winds](https://en.wikipedia.org/wiki/Castle_of_the_Winds) tiles and do the whole tile engine thing.

goals include:

* Allowing full affine transformations per tile, or at least for the "sprites" if doing it for all tiles is too expensive in terms of data transfers into the GPU.

**Comment by Nopey** [comment]
I'm curious what the motivation for full affine transformations on tiles is.

Any small set of tiles that are doing a little dance (say, dark wizard's summoning circle doing a wavy wobble) seem best suited for sprites,
while any large scale animation (say, an earthquake, the ocean, or the world slowly disintegrating into the void) seem like they'd be a great application for shaders

**Comment by Lokathor** [comment]
Yeah if it's too expensive to do for all tiles then it can be limited to just the "sprites" of the scene.
